doc/rados/configuration/pool-pg.conf

```diff
@@ -10,12 +10,13 @@
 	osd_pool_default_size = 3  # Write an object 3 times.
 	osd_pool_default_min_size = 2 # Accept an I/O operation to a PG that has two copies of an object.
 
+	# Disable autoscaling for manual management of placement groups.
+	osd_pool_default_pg_autoscale_mode = off
+
```

Comment:
I think disabling this here is counter to the party line.

Reply:
:joy: removed it, just wanted to draw more attention to the autoscaling, but the rest of changes are sufficient for that, so all good